sample_data_config/web-api/web-api-data-pipeline.config.yaml

```diff
@@ -20,16 +20,63 @@ webApi:
         - items
   #elifescience profile api
   - dataset: '{ENV}'
-    table: atest_profile_api
+    table: test_people_api
     dataUrl:
-      urlExcludingConfigurableParameters: https://api.elifesciences.org/profiles?
+      urlExcludingConfigurableParameters: https://api.elifesciences.org/people?
       configurableParameters:
         pageSizeParameterName: per-page
         defaultPageSize: 100
         pageParameterName: page
     response:
       itemsKeyFromResponseRoot:
         - items
+  # civi export
+  - dataset: '{ENV}'
+    table: civi_data_export
+    stateFile:
+      bucketName: '{ENV}-elife-data-pipeline'
+      objectName: 'airflow-config/generic-web-api/{ENV}-civi-export-processing-state.json'
+    urlSourceType:
+      name: 'civi'
+      typeSpecificVal:
+        fieldsToReturn:
+          - external_identifier
+          - tag
+          - group
+          - contact_type
+          - contact_sub_type
+          - last_name
+          - first_name
+          - middle_name
+          - email,source
```

Comment:
what does comma separated values do?

(I am trying to use the config as a starting point to understand the change better)

Reply:
acually, the natural form is comma separated values as this is how it is presented to the api, however, i decided to use just a list here for clarity in presentation. So, if you think this should be entered as "csv",  ur opionion will definitinely sway my decision

Reply:
Just looking at the configuration I am not sure what this is doing or why we need to configure it.
Do you mean all of the values are going to end up comma separated? Should in this case `email` and `source` appear as separate fields?
(I don't have any opinion either way yet)

Reply:
I think I only realise now that these are request arguments.

Reply:
> Just looking at the configuration I am not sure what this is doing or why we need to configure it.
> Do you mean all of the values are going to end up comma separated? Should in this case `email` and `source` appear as separate fields?
> (I don't have any opinion either way yet)

`- email,source` is a typo but it won't have which does not have bug-like effect on the data pipeline as I have to concatenate the list as comma seperated values in the web servcice request argument